dim/doc/api.rst

```diff
@@ -541,6 +554,14 @@ Block Functions
    delegation.
 
 
+.. function:: ipblock_move_to(cidr, block, layer3domain, to_layer3domain[, options])
+
+    Move block from source layer3domain *layer3domain* to
+    target layer3domain *to_layer3domain*.
```

Comment:
Maybe the following is a bit more detailed in the description

```
`ipblock_move_to` moves a container from *layer3domain* to the new layer3domain *to_layer3domain*.
IP addresses that are not part of a pool are moved too.
The container must have a parent. All subnets contained in the container get reassigned to the parent container.
If not present new views for the affected reverse zones will be created with the name of the layer3domain *to_layer3domain*. Entries for the IP addresses will then be deleted in the old view and recreated in the new view.

This operation can take a long time to succeed depending on the size of the container, the number of subnets and the number of IPs allocated in the container. In this time any write actions against the container, subnets or reverse zones may get blocked at the database level.
```

Reply:
Yes, thanks.
Having a more detailed description is much better, especially in this case.
I've updated the description according to your suggestion.